api/app/responses/api.py

```diff
@@ -458,3 +463,80 @@ def get(self, event_id):
         num_reviewers = review_config.num_reviews_required + review_config.num_optional_reviews if review_config is not None else 1
 
         return ResponseDetailAPI._serialize_response(response, language, review_form_id, num_reviewers)
+
+class ResponseExportAPI(restful.Resource):
+
+    def get(self):
+
+        def _get_answer(id, answers):
+            # Get the answer for a question
+            for a in answers:
+                if a.question_id == id:
+                    return a
+
+            return None
+
+        def _get_files(application_form, answers): 
+            # Get the response files that should be exported in the ZIP file
+
+            file_names = []
+
+            for section in application_form.sections:
+                if not section.questions:
```

Comment:
This check shouldn't be necessary, I don't think section.questions can be None, and if it's an empty list the for loop will just be skipped.